### ISSUE TITLE
About loss_loss_fn_kd

### ISSUE THREAD

**Issue by skyline-cpu** [DeepFakeIL/DFIL#16]
`def loss_fn_kd(y, labels, teacher_scores, T, alpha):
    return nn.KLDivLoss()(F.log_softmax(y/T,dim=1), F.log_softmax(teacher_scores/T,dim=1)) * (T*T * 2.0 * alpha) + F.cross_entropy(y, labels) * (1. - alpha)`  Is there something wrong with the loss function here? The teacher's output shouldn't be passed through log, right?
